### ISSUE TITLE
If import fails, the database and user are not removed

### ISSUE THREAD

**Issue by djavorszky** [djavorszky/ddn#28]
Steps to reproduce:
1. Get a dump
2. Call import-database with:
```
curl -X POST -d '{"database_name":"testdb", "username":"testuser", "password":"test", "dumpfile_location":"http://localhost/testdump.sql"}' localhost:7000/import-database
```
3. Call import-database again with the same query

Observed:
:x: Fails with message `{"Status":500,"Message":"database 'testdb' already exists"}`
Expected:
:white_check_mark: database and user should've been removed after failed import, hence the error should not be encountered.

Need a fix for all databases.

**Comment by djavorszky** [comment]
Resolved by df78cf1

**Comment by djavorszky** [comment]
Fixed, for sure this time
